lib/imports/api/authority-registry/server/methods.js

```diff
@@ -0,0 +1,145 @@
+import { check, Match } from 'meteor/check';
+import { authorityRegistryColl } from './collection';
+import { ValidatedMethod } from 'meteor/mdg:validated-method';
+
+const HTTPS_PORT = 443;
+
+/*
+Authority schema looks like this
+{
+ _id: 'abc123',
+ ip: '192.168.2.1',
+ port: 2976,
+ docIds: ['xyz321','foo837','bar221'],
+ recentFailedConnectionTimes: [13662212879613, 13662212879721, 1366221289532]
+}
+*/
+export const insertAuthority = new ValidatedMethod({
```

Comment:
I've added documentaiton to all methods and changed names for better clarity of authority vs authority server, coming soon